sqlx-core/src/logging.rs

```diff
@@ -8,6 +8,7 @@ macro_rules! log_execution {
         let elapsed = timer.elapsed();
         if elapsed >= std::time::Duration::from_secs(1) {
```

Comment:
While we're looking at this part of the code, can we lift this to a `const` at the top of the file so it's easier to find and change later if people are getting annoyed by the warnings?

As for name, maybe `SLOW_QUERY_THRESHOLD` or something.

Reply:
I didn't realize this was actually in a macro, so you'd need to refer to it by `crate::logging::SLOW_QUERY_THRESHOLD` but I think it's still better that way.